examples/Linux/lwip-include/arch/cc.h

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2001-2003 Swedish Institute of Computer Science.
```

Comment:
I don't actually know, but do we have to add wolfSSL Inc. boilerplate stuff here and elsewhere?

Reply:
@anhu not if we didn't change it.  If someone else wrote it, we have to keep their copyright notice on it, but if we change it, we have to _add_ our copyright.

Reply:
Got it...thank you @douzzer !!

Reply:
Correct, this code is directly copied from the lwIP source tree, but due to the way lwIP compiles it needs to be in our tree.